src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carCost = 40;
+  const days1 = carCost * days;
+  const days20 = carCost * days - 20;
+  const days50 = carCost * days - 50;
```

Comment:
1) give this vars more descriptive names such as: 
```suggestion
  const total = ...
  const basicDiscount = ...
  const highDiscount = ...
```
2) just create a var where you put the value of 20, for 50 – the same way